guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
+{% for item in page.imagesEntryScreens %}
+
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
 %}
+{% endfor %}
+
+</div>
+
+## Manual payment initiation
+
+When responding an invoice that contains all relevent information, the user can quickly review and approve it. In other scenarios, it may be required to manually enter or edit various details.
 
-Once you have gotten the address, it's time to enter the payment details. Bitcoin transactions are irreversible, so both the sender and receiver should take great care in correctly sharing and inputting addresses.
+<div class="image-slide-gallery">
 
-**QR Code** -- Camera access will need to be granted to your Bitcoin application to enable scanning of QR Codes. Once the camera detects a valid address in the QR Code, it should automatically fill the address field.
+{% for item in page.imagesInfo %}
 
-**Copy/Paste** -- When the sender receives the address or payment link as text, your application can detect a valid address in the clipboard and prompt the sender to press a button to paste it.
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
+%}
+{% endfor %}
 
 </div>
 
-**Do's**
+**Address**
+
+An address is a piece of information that bitcoin can be sent to, and that doesn’t change. Here are the common formats:
+- On-chain address
+- Lightning node ID
+- Lightning address
 
-- Indicate clearly if the address is valid or not
-- Show the full address if possible to help the sender visually verify it is correct
-- If space is a constraint, truncate the address in the middle so that both the beginning and end are visible
+Not that there are also static [invoice types]({{ '/guide/payments/send/payment-request-formats/' | relative_url }}) that can receive payments repeatedly. These are less intuitive overall due to their appearance, but could also be considered payment endpoints.
 
-**Don'ts**
+**Amounts**
 
-- Make it possible that a transaction is sent if the address is invalid
+If no amount is provided via a payment request, manual entry should be simple and convenient so users don’t accidentally send an incorrect amount. The amount should be displayed in both bitcoin or satoshi value, as well as the user’s local currency. Options to quickly toggle between them should be available. More on the [Units & Symbols page]({{ '/guide/payments/units-and-symbols/' | relative_url }}).
 
-## Inputting an amount
+## Fees
```

Comment:
This section should have some content around how a user could interact with fees (changing them for on-chain for example).

Reply:
Added a visual for a fee picker.

Reply:
Yew, looks good!